django/contrib/admin/bin/compress.py

```diff
@@ -28,8 +28,8 @@ def main():
     parser.add_argument("-q", "--quiet", action="store_false", dest="verbose")
     options = parser.parse_args()
 
-    compiler = closure_compiler if closure_compiler else os.path.expanduser(options.compiler)
-    if not os.path.exists(compiler):
+    compiler = Path(closure_compiler) if closure_compiler else Path(options.compiler).expanduser()
```

Comment:
Oh, I see the expansion is happening somewhere before here. e.g. passing `-c ~/bin/compiler.jar` results in `options.compiler` giving the full path. Is this happening in Python or in bash?